src/openms/source/METADATA/PeptideIdentification.cpp

```diff
@@ -134,6 +134,11 @@ namespace OpenMS
     hits_ = hits;
   }
 
+  void PeptideIdentification::setHits(std::vector<PeptideHit>&& hits)
+  {
+    hits_ = hits;
```

Comment:
```suggestion
    hits_ = std::move(hits);
```

otherwise this doesnt actually help?